src/java.base/share/classes/jdk/internal/event/X509CertificateEvent.java

```diff
@@ -31,6 +31,15 @@
  */
 
 public final class X509CertificateEvent extends Event {
+    private final static X509CertificateEvent EVENT = new X509CertificateEvent();
```

Comment:
Use blessed modifiers order
```suggestion
    private static final X509CertificateEvent EVENT = new X509CertificateEvent();
```